app/build.gradle

```diff
@@ -35,7 +35,7 @@ android {
 
     defaultConfig {
         applicationId "io.dolby.rtsviewer"
-        minSdk 25
```

Comment:
Can we use an alternative to DateTimeFormatter? The SDK actually does support minSDK 25

Reply:
Done used threeten as we used before